packages/ruleset/test/array-of-arrays.test.js

```diff
@@ -211,12 +211,11 @@ describe('Spectral rule: array-of-arrays', () => {
     it('Array of array of ints used within operation-level referenced parameter', async () => {
       const testDocument = makeCopy(rootDocument);
 
-      testDocument.components.parameters = {
-        ArrayParam: {
-          name: 'array_param',
-          in: 'header',
-          schema: arrayOfArrayOfInt
-        }
+      testDocument.components.parameters['ArrayParam'] = {
```

Comment:
Needed to be a bit more surgical since the root document now has other named parameters defined.